C/dag.h

```diff
@@ -364,17 +365,17 @@ simplicity_err verifyCanonicalOrder(dag_node* dag, const size_t len);
 /* This function fills in the 'WITNESS' nodes of a 'dag' with the data from 'witness'.
  * For each 'WITNESS' : A |- B expression in 'dag', the bits from the 'witness' bitstring are decoded in turn
  * to construct a compact representation of a witness value of type B.
- * This function only returns 'SIMPLICITY_NO_ERROR' when exactly 'witness.len' bits are consumed by all the 'dag's witness values.
- * If extra bits remain, then 'SIMPLICITY_ERR_WITNESS_TRAILING_BITS' is returned.
- * If there are not enough bits, then 'SIMPLICITY_ERR_WITNESS_EOF' is returned.
+ * If there are not enough bits, then 'SIMPLICITY_ERR_BITSTREAM_EOF' is returned.
```

Comment:
4c77c31217a590cc933c0e27570704ec59c45cf1: Should be `SIMPLICITY_ERR_WITNESS_EOF`